ARTICLE.md

```diff
@@ -1,59 +1,144 @@
-# Introduction to property based testing
-Mankind has evolved - perhaps it's time for how we test software to evolve as well?
+---
+title: From 10 to 1,000,000 test cases in under an hour: A beginner's guide to property-based testing 
```

Comment:
I wonder if we should change this to `1 to 100`... It's less dramatic, but those are the numbers we actually work with in the guide 😄 

Wdyt @fornwall?

Reply:
Fine with me!

Reply:
Regarding "1 to 100" - we have a settings(max_examples=500), and could bump that to say 10,000 or similar? Because 10,000 may be more showing "that's more than you could ever write yourself" than 100?